tests/media/server/gstplayer/genericPlayer/GstGenericPlayerTest.cpp

```diff
@@ -266,3 +266,12 @@ TEST_F(GstGenericPlayerTest, shouldReturnVolume)
     EXPECT_TRUE(m_sut->getVolume(resultVolume));
     EXPECT_EQ(resultVolume, kVolume);
 }
+
+TEST_F(GstGenericPlayerTest, shouldReturnMute)
+{
+    constexpr bool kMute{false};
+    bool resultMute{};
+    EXPECT_CALL(*m_gstWrapperMock, gstStreamVolumeGetMute(_)).WillOnce(Return(kMute));
```

Comment:
We should check that input volume is correct here. What I would do is create a new method in the test fixture similar to modifyContext in GstGenericPlayerPrivateTest so you can set the volume in the context and check the input here.